src/xmipp/libraries/reconstruction/project_tomography.cpp

```diff
@@ -138,6 +137,7 @@ void ProgProjectTomography::run()
 
         numProjs++;
         idx++;
+	angle+=projParam.tiltStep;
```

Comment:
similarly here, the `angle` is modified 60 lines deep in the loop (multiple occurrences)